cdap-ui/templates/oldToNewConfig.js

```diff
@@ -0,0 +1,93 @@
+var path = [
+  'cdap-etl-batch',
+  'cdap-etl-realtime',
+  'common'
+];
+
+var basePath = __dirname + '/';
+var fs = require('fs');
+
+path.forEach((folderName) => {
+  console.log('folderName: ', basePath + folderName);
```

Comment:
Ah crap. No this file should not be in the repo. Its a throw away tool just for converting the old json to new one. All the jsons are going to be in hydrator-plugins. This is just for the transition. Will remove this file.